docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+        python manage.py migrate &&
+        python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
```

Comment:
The `depends_on` directive is used to specify the dependency order of services. However, it does not wait for the `db` service to be fully ready before starting the `cinema` service. Consider using a script or tool to check the database readiness, as `depends_on` only ensures that the `db` service is started, not that it is ready to accept connections.